sgxcdr/dealer/App/ias-ra.c

```diff
@@ -179,22 +179,22 @@ void obtain_attestation_verification_report
 {
     CURL *curl;
     CURLcode res;
-  
+
     curl_global_init(CURL_GLOBAL_DEFAULT);
 
     curl = curl_easy_init();
     if(curl) {
         // curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
         curl_easy_setopt(curl, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_2);
-        curl_easy_setopt(curl, CURLOPT_URL, "https://test-as.sgx.trustedservices.intel.com:443/attestation/sgx/v2/report");
+        curl_easy_setopt(curl, CURLOPT_URL, "https://test-as.sgx.trustedservices.intel.com:443/attestation/sgx/v3/report");
```

Comment:
Is there value in having this hardcoded?

Reply:
Current SGX API attestation mechanism will be changed soon to use Intel DCAP for attestation service. So for now we will be hard coding the value and it will be changed as part of Intel DCAP migration.

Reply:
@suresh-marikkannu open an issue to track that and link it here.

Reply:
Following issue created to track the Intel DCAP based attestation migration,

[https://github.com/omec-project/c3po/issues/22#issue-419722431](url)